.github/workflows/reusable-upload.yml

```diff
@@ -89,7 +91,6 @@ jobs:
       - name: Publish to TestPyPI
         uses: pypa/gh-action-pypi-publish@release/v1
         with:
-          password: ${{ secrets.PYBATFISH_TEST_PYPI_TOKEN }}
           repository-url: https://test.pypi.org/legacy/
       - name: Set up Python
```

Comment:
FYI plan to refactor this out to remove testing steps that have write permissions, once the basic concept works.